cmd/collectors/ems/ems.go

```diff
@@ -275,10 +276,14 @@ func (e *Ems) getClusterTime() (time.Time, error) {
 				continue
 			}
 			clusterTime = t
-			break
+			dateOfEachNode = append(dateOfEachNode, currentClusterDate.String())
```

Comment:
Since we're already converting the string to time, it might be better to calculate the millisecond or seconds difference between each node and the first one. 

Accumulate that difference, and if it is greater than a predefined threshold, log the warning. That will allow some minor difference instead of requiring exact timestamps on each node.